src/sentry/auth/access.py

```diff
@@ -466,7 +466,7 @@ def from_member(
     return OrganizationMemberAccess(member, scopes, permissions)
 
 
-def from_auth(auth, organization: Organization) -> Access:
+def from_auth(auth, organization: Organization, scopes=[]) -> Access:
```

Comment:
@miruna1224 I think this doesn't need `scopes` parameter as it's un-used in this function.

Instead we can just pass `organization` on this line: https://github.com/getsentry/sentry/blob/34bb5f4c3909e671b6502eceeb849f9693d9794e/src/sentry/auth/access.py#L408

So the above SLOC should be `return from_auth(request.auth, organization)`

Reply:
You'll need to remove this default value. [Mutable types cannot be used a default values because python does bad things with them](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).